.eslintrc.js

```diff
@@ -48,13 +50,8 @@ module.exports = {
     {
       files: ['admin/src/**/*.js'],
       parser: '@babel/eslint-parser',
-      extends: [
-        'eslint:recommended',
-        'plugin:react/recommended',
-        'plugin:redux-saga/recommended',
```

Comment:
we dont use redux or redux-saga enywhere, so this should not be needed

Reply:
Yes makes sense